lib/Driver/ToolChains.cpp

```diff
@@ -231,6 +231,8 @@ static void addCommonFrontendArgs(const ToolChain &TC, const OutputInfo &OI,
                        options::OPT_enable_experimental_dependencies);
   inputArgs.AddLastArg(arguments,
                        options::OPT_experimental_dependency_include_intrafile);
+  inputArgs.AddLastArg(
```

Comment:
Ooo, is `inputArgs.AddLastArg` missing for `options::OPT_enable_experimental_differentiable_programming`? I'm not sure whether adding it is necessary, but it seems necessary.

Reply:
indeed it is missing and it seems to be necessary but since the option is not used anywhere in the code yet (27446)  I am not what the consequences are of not doing `inputArgs.AddLastArg`

Reply:
> Honestly, I am not sure what the consequences are of not doing `inputArgs.AddLastArg`.

Me neither 😮